Vagrantfile.example

```diff
@@ -8,24 +8,65 @@
 # vagrant up
 # vagrant ssh -c "cd /vagrant/; pserve development.ini --reload"
 
-Vagrant.configure(2) do |config|
-  config.vm.box_url = "https://download.fedoraproject.org/pub/fedora/linux/releases/24/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-24-1.2.x86_64.vagrant-libvirt.box"
-  config.vm.box = "f24-cloud-libvirt"
-  config.vm.network "forwarded_port", guest: 6543, host: 6543
-  config.vm.synced_folder ".", "/vagrant", type: "sshfs"
-
-  config.vm.provider :libvirt do |domain|
-      domain.cpus = 4
-      domain.graphics_type = "spice"
-      # The unit tests use a lot of RAM.
-      domain.memory = 4096
-      domain.video_type = "qxl"
-  end
-
-  # Ansible needs the guest to have these
-  config.vm.provision "shell", inline: "sudo dnf install -y libselinux-python python2-dnf"
-
-  config.vm.provision "ansible" do |ansible|
-      ansible.playbook = "devel/ansible/playbook.yml"
-  end
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+ config.vm.box_url = "https://download.fedoraproject.org/pub/fedora/linux/releases/24/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-24-1.2.x86_64.vagrant-libvirt.box"
+ config.vm.box = "f24-cloud-libvirt"
+
+ # Forward traffic on the host to the development server on the guest.
+ # You can change the host port that is forwarded to 5000 on the guest
+ # if you have other services listening on your host's port 80.
+ config.vm.network "forwarded_port", guest: 6543, host: 6543
+
+ # This is an optional plugin that, if installed, updates the host's /etc/hosts
+ # file with the hostname of the guest VM. In Fedora it is packaged as
+ # ``vagrant-hostmanager``
+ if Vagrant.has_plugin?("vagrant-hostmanager")
+     config.hostmanager.enabled = true
+     config.hostmanager.manage_host = true
+ end
+
+ # Vagrant can share the source directory using rsync, NFS, or SSHFS (with the vagrant-sshfs
+ # plugin). Consult the Vagrant documentation if you do not want to use SSHFS.
+ config.vm.synced_folder ".", "/vagrant", disabled: true
+ config.vm.synced_folder ".", "/vagrant", type: "sshfs", sshfs_opts_append: "-o nonempty"
```

Comment:
Why don't we go ahead and switch this to ```/home/vagrant/bodhi``` or something? I've been annoyed at working in ```/vagrant```…